EjerciciosDiaUno.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+
+public class Class1
+{
+	public Class1()
+	{
+        int Resultado = 0;
```

Comment:
variables inician en minuscula

```suggestion
        int resultado = 0;
```

Reply:
Realizare una reestructuración del Ejercicio # 1 de .Net a lo largo y ancho del código, nombramientos de clases, métodos y variables.